feat(*): update everything to v13 (#170)
AnIdiotsGuide/discordjs-bot-guide: frequently-asked-questions.md

// Set the bot's presence (activity and status)
client.on("ready", () => {
    client.user.setPresence({

>         activities: [

Reviewer: Why newline the array?
It just makes the code uglier...

client.on("ready", () => {
    client.user.setPresence({
        activities: [{ 

>           name: 'my code',
>           type: 'WATCHING'
>           }],

Reviewer 2: Why was the last line given more room here? Usually its in line with the top object.
```suggestion
          name: 'my code',
          type: 'WATCHING'
        }],
```

  // Now we get into the heavy stuff: first channel in order where the bot can speak
  // hold on to your hats!
  return guild.channels.cache

>    .filter(c => c.type === "TEXT" &&

Reviewer 3: ```suggestion
   .filter(c => c.type === "GUILD_TEXT" &&
```

```

```javascript

> // Fetching a message by ID (Discord.js versions 9 through 13)
> // note: you can line return right before a "dot" in JS, that is valid.
> message.channel.messages.fetch({ around: "352292052538753025", limit: 1 })
>   .then(messages => {
>     const fetchedMsg = messages.first(); // messages is a collection!
>     // Check if the author of the message is the bot
>     if (message.client.user.id !== fetchedMsg.id) return console.log('I\'m not the author of that message!');
> 
>     // Edit the message
>     fetchedMsg.edit("This fetched message was edited");
>   });

Reviewer 2: Any reason in particular we are using around? We can fetch directly by id. Also no reason to mention this works from v9 up cause this guide isn't for v9 up.

Not sure if we should keep the `// note: you can line return right before a "dot" in JS, that is valid.` comment. This guide states: `We do, however expect you to have a basic understanding of JavaScript before attempting to write bots.` I believe this is covered by the basic js knowledge and this isn't the only place we have a newline with a dot after it.
```suggestion
// Fetching a message by ID
// note: you can line return right before a "dot" in JS, that is valid.
message.channel.messages.fetch("352292052538753025")
  .then(message => {
    // do something with it
    // Check if the author of the message is the bot
    if (message.client.user.id !== message.id) return console.log('I\'m not the author of that message!');
    // Edit the message
    message.edit("This fetched message was edited");
  });
  ```
